metron-analytics/metron-profiler/src/main/assembly/assembly.xml

```diff
@@ -63,7 +63,7 @@
         <fileSet>
             <directory>${project.basedir}/target</directory>
             <includes>
-                <include>${project.artifactId}-${project.version}.jar</include>
+                <include>${project.artifactId}-${project.version}-uber.jar</include>
             </includes>
```

Comment:
Think I've got it.  Added some additional tests to the list in the PR description.  Will have another commit coming that updates the RPM spec file.